scripts/kubernetes/meta-standalone.yaml

```diff
@@ -1,4 +1,19 @@
 ---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: meta-service-pv-volume
+  labels:
+    type: local
+spec:
+  storageClassName: local-storage
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/data"
```

Comment:
Is it safe to share the same host path with other pods?

Reply:
> Is it safe to share the same host path with other pods?

I'm afraid not safe, change it to another maybe better.

Reply:
I have noticed that other services have similar problems. Are you willing to change them at the same time?

For naming, I prefer:

- meta: `/mnt/databend/meta`
- minio: `/mnt/databend/minio`
- component_name: `/mnt/databend/component_name`

Does this look good to you?

Reply:
> I have noticed that other services have similar problems. Are you willing to change them at the same time?
> 
> For naming, I prefer:
> 
> * meta: `/mnt/databend/meta`
> * minio: `/mnt/databend/minio`
> * component_name: `/mnt/databend/component_name`
> 
> Does this look good to you?

Yeah,mostly LGMT. BTW, where is the `component_name ` service? Or what is the `component_name ` mean?